chunjun-connectors/chunjun-connector-starrocks/pom.xml

```diff
@@ -12,56 +12,54 @@
 	<artifactId>chunjun-connector-starrocks</artifactId>
 	<name>ChunJun : Connectors : StarRocks</name>
 
+
+	<properties>
+		<maven.compiler.source>8</maven.compiler.source>
+		<maven.compiler.target>8</maven.compiler.target>
+		<arrow.version>5.0.0</arrow.version>
+	</properties>
+
 	<dependencies>
+		<dependency>
+			<groupId>mysql</groupId>
+			<artifactId>mysql-connector-java</artifactId>
+			<version>5.1.49</version>
+		</dependency>
+
+		<!--		<dependency>-->
+		<!--			<groupId>io.vertx</groupId>-->
+		<!--			<artifactId>vertx-core</artifactId>-->
+		<!--			<version>3.4.2</version>-->
+		<!--		</dependency>-->
+
+		<!--		<dependency>-->
+		<!--			<groupId>io.vertx</groupId>-->
+		<!--			<artifactId>vertx-web-client</artifactId>-->
+		<!--			<version>3.4.2</version>-->
+		<!--		</dependency>-->
+
+		<dependency>
+			<groupId>com.alibaba</groupId>
+			<artifactId>fastjson</artifactId>
```

Comment:
Duplicate dependencies are depended on the project, and we have other json-processed dependencies, like Gson, in the root pom.